src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/repository/CourtEventChargeRepository.java

```diff
@@ -14,7 +14,8 @@ public interface CourtEventChargeRepository extends CrudRepository<CourtEventCha
         "join cec.eventAndCharge.offenderCharge oc " +
         "join oc.offenderBooking ob " +
         "join ob.offender o " +
-        "where o.nomsId = :offenderId"
+        "where o.nomsId = :offenderId and " +
```

Comment:
`and` should really be on the same line as `exists`

Reply:
Done